patches/@react-navigation+native+6.1.12+002+fix-getting-history-entry-when-navigating-back.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@react-navigation/native/lib/module/useLinking.js b/node_modules/@react-navigation/native/lib/module/useLinking.js
+index 95a0e32..adb25a0 100644
+--- a/node_modules/@react-navigation/native/lib/module/useLinking.js
++++ b/node_modules/@react-navigation/native/lib/module/useLinking.js
+@@ -294,7 +294,7 @@ export default function useLinking(ref, _ref) {
+           try {
+             if (nextIndex !== -1 && nextIndex < currentIndex &&
+             // We should only go back if the entry exists and it's less than current index
```

Comment:
Hmm could you update the comments too? seems like this makes a bit less sense now after your change

Reply:
Hmm but it still makes sense

We should only go back if the entry exists -> `history.get(nextIndex)`

and it's less than current index -> `nextIndex < currentIndex`

The thing is `get` needs absolute index and `go` relative. The code assumed both use relative

Reply:
Oh yeah sorry, I saw this change only and it made me think the `less thank currentIndex` is off 

- history.get(nextIndex - currentIndex)) {
+            history.get(nextIndex)) {